html/gui/js/modules/dashboard/CenterReportCardComponent.js

```diff
@@ -121,7 +121,7 @@ XDMoD.Modules.SummaryPortlets.CenterReportCardPortlet = Ext.extend(Ext.ux.Portle
             },
             listeners: {
                 load: function () {
-                    // Make sure that once we're loaded we remove this portlets
+                    // Make sure that once we're loaded we remove this components
```

Comment:
```suggestion
                    // Make sure that once we're loaded we remove this component's
```
Might as well fix the grammar since this is being changed.